api/filter/http/gzip.proto

```diff
@@ -23,7 +25,7 @@ message Gzip {
   google.protobuf.UInt32Value content_length = 2 [(validate.rules).uint32.gte = 30];
 
   enum CompressionLevel {
-    DEFAULT = 0;
+    DEFAULT_LEVEL = 0;
```

Comment:
Could you actually move this/below inside a message and then call the enum `Enum` or something? (The standard C++ hack before enum class in C++11)? cc @gsagula